source/beamline/BioXAS/BioXASBeamStatus.cpp

```diff
@@ -18,6 +40,18 @@ BioXASBeamStatus::~BioXASBeamStatus()
 
 }
 
+bool BioXASBeamStatus::isConnected() const
+{
+	bool connected = (
+				shutters_ && shutters_->isConnected() &&
+				valves_ && valves_->isConnected() &&
+				mirrorMaskState_ && mirrorMaskState_->isConnected() &&
+				monoMaskState_ && monoMaskState_->isConnected()
```

Comment:
@helfrij I want to confirm the logic here. If we forgot to initialize the `mirrorMaskState_` and/or `monoMaskState_`. `isConnected()` will always return false. So far, both the main/side bioxas are using BioXASBeamStatus and initialized the mask control. Will imaging use `BioXASBeamStatus` ?